cargo-smart-release/src/changelog/write.rs

```diff
@@ -55,7 +55,12 @@ impl RepositoryUrl {
 
     fn cleaned_path(&self) -> String {
         let path = self.inner.path.to_str_lossy().into_owned();
-        path.strip_suffix(".git").map(ToOwned::to_owned).unwrap_or(path)
+        let path = path.strip_suffix(".git").map(ToOwned::to_owned).unwrap_or(path);
```

Comment:
Cargo Smart Release part is now in its own commit.
It might not have enough test coverage but it was enough to raise proper errors on those changes allowing me to detect and fix this part 👍🏼